server/src/main/java/com/objectcomputing/checkins/services/memberprofile/currentuser/CurrentUserController.java

```diff
@@ -60,6 +63,9 @@ public HttpResponse<CurrentUserDTO> currentUser(@Nullable Authentication authent
         String lastName = name != null ? name.substring(name.indexOf(' ') + 1).trim() : "";
 
         MemberProfile user = currentUserServices.findOrSaveUser(firstName, lastName, workEmail);
+        user.setLastSeen(LocalDate.now());
+        System.out.println(user.getLastSeen());
+        memberProfileServices.saveProfile(user);
```

Comment:
I'd consider adding a new update method to MemberProfileServices. This saveProfile method also makes a findBy call, which isn't needed since you already have the MemberProfile object.

Something like this
```
    @Override
    public MemberProfile updateProfile(MemberProfile memberProfile) {
        return memberProfileRepository.update(memberProfile);
    }
```

Reply:
Done. Ty for the suggestion.